easybuild/easyconfigs/v/Valgrind/Valgrind-3.15.0-foss-2020a.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'Valgrind'
+version = '3.15.0'
+
+homepage = 'https://valgrind.org'
+description = "Valgrind: Debugging and profiling tools"
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'optarch': False}
```

Comment:
@maxim-masterov Do you have a specific reason for disabling `optarch` here?

Reply:
No particular reason, it was transferred from the previous config files. I see it down to Valgrind-3.13 version. Do you have any idea why?

Reply:
See discussion in https://github.com/easybuilders/easybuild-easyconfigs/pull/3180, installation was failing back then, but maybe that's no longer an issue now with `optarch` enabled on recent processor architectures?

Reply:
Okay. I've checked with foss and it works just fine with enabled optarch. However, a compilation with intel fails. So, I'm keeping it disabled for intel.